### PR TITLE
Move partials to its own directory. 

### PR DESCRIPTION
**Pull request by vpereira**
In the #5854 I refactored the _sidelinks view, however I think that all partials should be packed together in it's own directory (to avoid clutter), named sidelinks

### CODE REVIEW

File: src/api/app/helpers/webui/packages/partials_path_helper.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module Webui::Packages::PartialsPathHelper
+  def partial_path(directory, partial)
+    File.join('webui2', 'webui', 'package', directory, partial)
```
Comment (dmarcoux) [inline src/api/app/helpers/webui/packages/partials_path_helper.rb]: I don't think we need to have code for this. It's already possible to achieve this by specifying the full path to the partial starting from `app/views`. This is what this code does too

Comment (dmarcoux) [reply src/api/app/helpers/webui/packages/partials_path_helper.rb]: After talking with @vpereira, I am also fine with this approach. I prefer not having an helper, but it's not a issue to have it too. I won't block this PR due to this.

File: src/api/app/views/webui2/webui/package/_side_links.html.haml [hunk 2]
```diff
@@ -1,24 +1,25 @@
--# FIXME: refactor this file
 %ul.list-unstyled
   - if failures > 0
-    = render partial: 'show_failures', locals: { failures: failures, package_name: package.name, project: project }
+    = render partial: partial_path('side_links', 'show_failures'), locals: { failures: failures, package_name: package.name, project: project }
```
Comment (hennevogel) [inline src/api/app/views/webui2/webui/package/_side_links.html.haml]: I do take issue with this. I know what `render partial: 'directory/subdirectory/file'` does, I have to loop up the meaning and implementation of `partial_path`. It's code we do not need, which is the worst kind of code. Sorry...

Comment (vpereira) [reply src/api/app/views/webui2/webui/package/_side_links.html.haml]: TL;DR
my bikeshed is green :alien:

actually, as soon as we get rid of ```webui/webui2``` is easier to change in one place than switch it  everywhere, we could put it in a variable, but that's exactly why helpers are there @hennevogel

Comment (bgeuken) [reply src/api/app/views/webui2/webui/package/_side_links.html.haml]: But once we switched to webui2 we would drop the `partial_path` or not? And is this planned to be used only for package partials or others as well?

Comment (vpereira) [reply src/api/app/views/webui2/webui/package/_side_links.html.haml]: I would use it as well in the #5867 (huge paths are coming XD) but I removed it already. Thank you @bgeuken <3

Comment (hennevogel) [reply src/api/app/views/webui2/webui/package/_side_links.html.haml]: > my bikeshed is green

If I would think this is not important I would mark my comment as nitpick. I haven't :cowboy_hat_face:

**Pull request merged**